docs/Basic-Guide.md

```diff
@@ -41,22 +41,7 @@ In order to use ML-Agents within Unity, you need to change some Unity settings f
 
 ![Running a pretrained model](images/running-a-pretrained-model.gif)
 
-## Building an Example Environment
-
-The first step is to open the Unity scene containing the 3D Balance Ball
-environment:
-
-1. Launch Unity.
-2. On the Projects dialog, choose the **Open** option at the top of the window.
-3. Using the file dialog that opens, locate the `unity-environment` folder 
-within the ML-Agents project and click **Open**.
-4. In the **Project** window, navigate to the folder 
-`Assets/ML-Agents/Examples/3DBall/`.
-5. Double-click the `3DBall` file to load the scene containing the Balance 
-Ball environment.
-
-![3DBall Scene](images/mlagents-Open3DBall.png)
-
+## Setting the Brain to External
 Since we are going to build this environment to conduct training, we need to 
 set the brain used by the agents to **External**. This allows the agents to 
 communicate with the external training process when making their decisions.
```

Comment:
Now that the in-editor training is no longer needing a build of executable, We should incorporate this section as a subsection into "Training the Brain with Reinforcement Learning"